app/models/invoice.js

```diff
@@ -98,6 +98,26 @@ export default AbstractModel.extend(DateFormat, NumberFormat, {
 
   patientResponsibilityTotals: Ember.computed.mapBy('lineItems', 'amountOwed'),
   patientResponsibility: Ember.computed.sum('patientResponsibilityTotals'),
+  finalPatientResponsibility: function() {
```

Comment:
Use
```js
finalPatientResponsibility: computed('patientResponsibility', 'paymentProfile', function() {
...
})
``` 
instead of
```js
finalPatientResponsibility: function() {
...
}.property('patientResponsibility', 'paymentProfile'),
```